applications/openshift/kubelet/kubelet_configure_tls_min_version/kubernetes/shared.yml

```diff
@@ -0,0 +1,19 @@
+---
+# platform = multi_platform_ocp
+apiVersion: machineconfiguration.openshift.io/v1
+kind: KubeletConfig
+metadata:
+  annotations:
+    complianceascode.io/node-role: "{{.var_role_worker}}"
+spec:
+  kubeletConfig:
+    tlsMinVersion: VersionTLS12
```

Comment:
Wondering if you could create a new variable, and replace `VersionTLS12` to that variable, so a user can specify which version to use using a tailored profile.

Reply:
comment incorporated.